test/fmf/tests/fdo-bootc.sh

```diff
@@ -0,0 +1,260 @@
+    if [[ $VM_STATUS == running ]]; then
+        sudo virsh destroy "${VM_NAME}"
+    fi
+    sudo virsh undefine "${VM_NAME}" --nvram
+    # TO DO: Remove tag from quay.io registry
+    # skopeo delete --creds "${QUAY_USERNAME}:${QUAY_PASSWORD}" "${QUAY_REPO_URL}:${QUAY_REPO_TAG}"
+}
+
+# Colorful output.
+function greenprint {
+    echo -e "\033[1;32m${1}\033[0m"
+}
+
+wait_for_ssh_up () {
+    SSH_STATUS=$(sudo ssh "${SSH_OPTIONS[@]}" -i "${SSH_KEY}" "${SSH_USER}@${1}" '/bin/bash -c "echo -n READY"')
+    if [[ $SSH_STATUS == READY ]]; then
+        echo 1
+    else
+        echo 0
+    fi
+}
+
+check_result () {
+    greenprint "🎏 Checking for test result"
+    if [[ $RESULTS == 1 ]]; then
+        greenprint "💚 Success"
+    else
+        greenprint "❌ Failed"
+        clean_up
+        exit 1
+    fi
+}
+
+trap clean_up EXIT
+
+sudo dnf install -y \
+    cargo \
+    openssl \
+    openssl-devel \
+    git \
+    make \
+    systemd \
+    krb5-devel \
+    python3-docutils \
+    gpgme-devel \
+    libassuan-devel \
+    systemd-rpm-macros \
+    rpmdevtools \
+    golang \
+    go-rpm-macros \
+    python3-devel \
+    selinux-policy-devel \
+    device-mapper-devel \
+    podman \
+    qemu-img \
+    qemu-kvm \
+    libvirt-client \
+    libvirt-daemon-kvm \
+    libvirt-daemon \
+    virt-install
+
+# Clone upstream repo and build fdo packages
+echo "Build fdo packages"
+cd /tmp/"${REPO_NAME}"
+make rpm
+sudo dnf -y install rpmbuild/RPMS/x86_64/*
+cd -
+mkdir -pv packages
+sudo cp /tmp/"${REPO_NAME}"/rpmbuild/RPMS/x86_64/* ./packages
+
+# Manufacturing server setup
+DOWNLOAD_NODE="${DOWNLOAD_NODE}" ./setup-manufacturing-server.sh
+
+# Login to Stage registry
+sudo podman login -u "${STAGE_REDHAT_IO_USERNAME}" -p "${STAGE_REDHAT_IO_TOKEN}" registry.stage.redhat.io
+
+# Register the system to be able to install missing packages
+sudo subscription-manager register --username "${REDHAT_IO_USERNAME}" --password "${REDHAT_IO_TOKEN}"
+
+# Prepare Containerfile
+tee Containerfile > /dev/null << STOPHERE
+FROM registry.stage.redhat.io/rhel10/rhel-bootc:10.0
+RUN echo 'root' | passwd --stdin root
+# Copy the local RPM files into the container
+COPY ./packages/* /tmp/
+# Install packages
+RUN dnf install -y \
+    /tmp/fido-device-onboard-debuginfo-*.el10.x86_64.rpm \
+    /tmp/fido-device-onboard-debugsource-*.el10.x86_64.rpm \
+    /tmp/fdo-init-debuginfo-*.el10.x86_64.rpm \
+    /tmp/fdo-init-*.el10.x86_64.rpm \
+    /tmp/fdo-client-debuginfo-*.el10.x86_64.rpm \
+    /tmp/fdo-client-*.el10.x86_64.rpm \
+    clevis \
+    clevis-dracut \
+    clevis-luks \
+    clevis-pin-tpm2 \
+    clevis-systemd
+RUN systemctl enable fdo-client-linuxapp.service
+STOPHERE
+
+# Create container image
+sudo podman build -f Containerfile -t quay.io/"${QUAY_USERNAME}"/test-repository/rhel10-bootc:latest
+
+# Push to quay.io
+sudo podman push rhel10-bootc --creds="${QUAY_USERNAME}":"${QUAY_PASSWORD}" quay.io/"${QUAY_USERNAME}"/test-repository/rhel10-bootc:latest
```

Comment:
I think we should also do `skopeo --delete quay.io....`  to cleanup tags at the end of the process

Reply:
Done.